packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapController.m

```diff
@@ -250,15 +250,15 @@ - (void)onMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result {
     result(nil);
   } else if ([call.method isEqualToString:@"map#takeSnapshot"]) {
     if (self.mapView != nil) {
-      UIGraphicsImageRendererFormat *format = [UIGraphicsImageRendererFormat defaultFormat];
-      format.scale = [[UIScreen mainScreen] scale];
       UIGraphicsImageRenderer *renderer =
-          [[UIGraphicsImageRenderer alloc] initWithSize:self.mapView.frame.size format:format];
-
+          [[UIGraphicsImageRenderer alloc] initWithSize:self.mapView.bounds.size];
+      // The API originally used doesn't work in ios 17
```

Comment:
can you provide more details other than just "doesn't work"? Also please explicitly put the original API here (we can't tell what "The API originally used" is after you remove it.

Reply:
Yep ! Is it enough right now ?